rpc/audit/rpcbinding.go

```diff
@@ -0,0 +1,1591 @@
+// Package audit contains RPC wrappers for NeoFS Audit contract.
+package audit
+
+import (
+	"crypto/elliptic"
+	"errors"
+	"fmt"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/neorpc/result"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/unwrap"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+	"math/big"
+	"unicode/utf8"
+)
+
+// AuditauditHeader is a contract-specific audit.auditHeader type used by its methods.
+type AuditauditHeader struct {
+	epoch *big.Int
+	cid []byte
+	from *keys.PublicKey
+}
+
+// CommonBallot is a contract-specific common.Ballot type used by its methods.
+type CommonBallot struct {
+	ID []byte
+	Voters keys.PublicKeys
+	Height *big.Int
+}
+
+// LedgerBlock is a contract-specific ledger.Block type used by its methods.
+type LedgerBlock struct {
```

Comment:
we have this and similar called types in each package of the same level. Could u pls explain why do we need them to be exported? 

btw there are also more contract-specific ones like `AuditauditHeader`.

Reply:
It's nspcc-dev/neo-go#3071, they'll be removed once it's solved. We can live with some garbage in these for now.